CHANGELOG

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.9.3] - 2019-01.25
```

Comment:
The previous change had an error in the date string. Could you change them both to dash-separated?

Reply:
Oh, man! Of course! :)